src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write code here
+const treeItems = document.querySelectorAll('.tree li');
+
+treeItems.forEach((item) => {
+  const nestedList = item.querySelector('ul');
+
+  if (nestedList) {
+    const span = document.createElement('span');
+
+    span.textContent = item.childNodes[0].nodeValue.trim();
+    item.childNodes[0].replaceWith(span);
```

Comment:
Replacing the first child node with a span might cause issues if the first child is not a text node. Ensure that the first child is indeed a text node before replacing it, or handle other types of nodes appropriately.